shell/src/main/java/alluxio/cli/fsadmin/command/CheckpointCommand.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.command;
+
+import alluxio.Constants;
+import alluxio.cli.CommandUtils;
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.exception.status.InvalidArgumentException;
+
+import org.apache.commons.cli.CommandLine;
+
+import java.io.IOException;
+import java.util.Timer;
+import java.util.TimerTask;
+
+/**
+ * Command for triggering a checkpoint in the primary master journal system.
+ */
+public class CheckpointCommand extends AbstractFsAdminCommand {
+
+  /**
+   * @param context fsadmin command context
+   * @param alluxioConf Alluxio configuration
+   */
+  public CheckpointCommand(Context context, AlluxioConfiguration alluxioConf) {
+    super(context);
+  }
+
+  @Override
+  public String getCommandName() {
+    return "checkpoint";
+  }
+
+  @Override
+  public int run(CommandLine cl) throws IOException {
+    Timer timer = new Timer();
+    timer.schedule(new TimerTask() {
+        @Override
+        public void run() {
+          mPrintStream.println("Checkpointing may take a while.");
```

Comment:
this is pretty verbose - in other places we've logged a single "." every few seconds without adding newlines, see https://github.com/Alluxio/alluxio/blob/master/job/client/src/main/java/alluxio/client/job/JobGrpcClientUtils.java#L97

Reply:
Used the `...`  interpreting waiting